genetlink/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "genetlink"
-version = "0.1.0"
+version = "0.2.0"  # TODO: drop this comment - already bumped version for trait changes
```

Comment:
I think you forgot to remove those comments?

Reply:
I wanted to make sure the affected packages get "major" bumps; the idea was that the comments get removed when actually releasing (instead of doing minor bumps).

If you prefer doing it a different way just let me know.

Reply:
Oh I see. That's fine then I'll remove them when releasing.